Godeps/Godeps.json

```diff
@@ -110,7 +110,7 @@
 		},
 		{
 			"ImportPath": "github.com/letsencrypt/go-jose",
-			"Rev": "77671e91fe89678878df4bde9e6b653a0d190dfb"
+			"Rev": "63931e513ab97b230f56ab0b49dc221399d44a23"
```

Comment:
The letsencrypt/go-jose master's HEAD is now 15b73d6f0363b256ddb080db1c64563064a4e773. Mind doing the `godep update` one more time to get it there?